_articles/organizations/getting-started-organizations.md

```diff
@@ -0,0 +1,196 @@
+---
+layout: article
+title: Getting Started with Bitwarden Organizations
+categories: [organizations]
+featured: false
+popular: false
+hidden: true
+tags: [tutorial]
+---
+
+With the Bitwarden free account, and a free Organization for up to 2 users, you can experience secure sharing in no time!
+
+- [Introduction](#introduction)
+  * [What are organizations?](#what-are-organizations)
+- [Set up the First and Second User Accounts](#set-up-the-first-and-second-user-accounts)
+  * [1. Create an account for User 1](#1-create-an-account-for-user-1)
+  * [2. Verify Your Email](#2-verify-your-email)
+  * [3. Set up the Second User Account](#3-set-up-the-second-user-account)
+- [Setup and Share with Organizations](#setup-and-share-with-organizations)
+  * [4. Create a new Organization](#4-create-a-new-organization)
+  * [5. Add the Second User to your Organization](#5-add-the-second-user-to-your-organization)
+  * [6. Accepting the Invitation](#6-accepting-the-invitation)
+  * [7. Sharing within an Organization](#7-sharing-within-an-organization)
+- [Continuing with Bitwarden](#continuing-with-bitwarden)
+
+# Introduction
+Using a password manager makes it easy to store and access unique and secure passwords across all of your devices. This keeps your online accounts safer than repeating passwords or leaving them exposed in unencrypted formats like spreadsheets, documents, or sticky notes!
+
+To get even more out of your password manager, you can collaborate and share with others. At work, you may choose to share common logins to office wifi networks and online services, or a shared company credit card, or a set of secure notes. Personally you may choose to share a few passwords with a significant other.
+
+Bitwarden enables all types of password and secure sharing through Organizations. 
+
+If you would like to try it out, Bitwarden has a free account for individuals, and the ability to set up a free Organization for up to two users and two Collections of secure information.
+
+This tutorial walks you through the setup for two basic accounts and sharing in an Organization.
+
+## What are organizations?
+A Bitwarden organization is an entity that relates users together that want to share items. An organization could be a family, team, company, or any other type of group that desires to share items in Bitwarden.
+
+For more, please see [What are organizations?](https://help.bitwarden.com/article/what-is-an-organization/)
+
+**Are Organizations Different than a Premium Membership?**
+Yes. Organizations are for sharing items across multiple users
```

Comment:
Is this supposed to be a separate paragraph than the content below? Needs a period to end the sentence, right?